carp.common/src/commonMain/kotlin/dk/cachet/carp/common/application/data/NoData.kt

```diff
@@ -0,0 +1,11 @@
+package dk.cachet.carp.common.application.data
+
+import kotlinx.serialization.Serializable
+
+
+/**
+ * Placeholder for generic `Data` types to indicate there is no associated data.
+ * This should not be serialized; instead, nullable `Data` should be used.
+ */
+@Serializable
+object NoData : Data
```

Comment:
Maybe put this in the same file as `Data` - doesn't matter much, it just seems very old-school Java-like :)

Reply:
Makes perfect sense! I also didn't add it in the docs.